core/reporter-impl/src/test/java/org/arquillian/reporter/impl/base/AbstractReporterTestBase.java

```diff
@@ -102,6 +103,7 @@ private void registerBuilders() {
     private void initiateStringKeys() {
         ArrayList<StringKey> stringKeys = new ArrayList<>();
         stringKeys.add(new ReporterCoreKey());
+        stringKeys.add(new TestExtensionStringKey());
```

Comment:
This should be placed here: https://github.com/arquillian/arquillian-reporter/pull/18/files#diff-201783225109103171e4bf201ba27b33R76